db/db_test.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) The EfficientGo Authors.
+// Licensed under the Apache License 2.0.
+
+package e2edb
+
+import (
+	"testing"
+
+	"github.com/efficientgo/core/testutil"
+	"github.com/efficientgo/e2e"
+)
+
+func TestMinio(t *testing.T) {
+	t.Parallel()
+
+	e, err := e2e.New()
+	testutil.Ok(t, err)
+	t.Cleanup(e.Close)
+
+	minio := NewMinio(e, "mintest", "bkt")
+	testutil.Ok(t, e2e.StartAndWaitReady(minio))
```

Comment:
Maybe we should do some operation here, to validate that `StartAndWaitReady` returning actually means that minio is ready to receive requests?

Reply:
Done at https://github.com/efficientgo/e2e/pull/58/commits/aa6fa59a54d16f3abeb03d45340d7dd569c02b15